src/service/defaultStructure.service.ts

```diff
@@ -32,11 +32,12 @@ export class DefaultStructureService {
    * @param {string} [functionName=''] - creates the necessary files for the init command
    * @memberof DefaultStructureService
    */
-  public create(functionName = 'exampleFunction', update?: boolean): void {
+  public create(functionName?, update?: boolean): void {
```

Comment:
Missing type declaration

```suggestion
  public create(functionName?: string, update?: boolean): void {
```